### PR TITLE
Moved physics constants from Cello layer to Enzo layer

### PR DESCRIPTION
**Pull request by mabruzzo**
As we discussed this morning I moved the Physics constants from the Cello layer to the `Enzo` layer.

I went a little off script here and put the constants into a new namespace called `enzo_constants`. If people would prefer otherwise, I can easily put them into the existing `enzo` namespace

~EDIT:I didn't compile the code first. There's a few things I need to fix~ It compiles now

### CODE REVIEW

**Comment by jwise77** [comment]
It's not compiling on CI because there are some constants in Cello
```
../src/Cello/problem_Units.hpp:102:20: error: ‘mass_hydrogen’ is not a member of ‘cello’
  102 |   { return (cello::mass_hydrogen)*std::pow(velocity(),2)/(cello::kboltz); }
      |                    ^~~~~~~~~~~~~
../src/Cello/problem_Units.hpp:102:66: error: ‘kboltz’ is not a member of ‘cello’
  102 |   { return (cello::mass_hydrogen)*std::pow(velocity(),2)/(cello::kboltz); }
  ```

**Comment by mabruzzo** [comment]
Thanks! I just fixed that issue.

**Review by stefanarridge — APPROVED** [review]

File: src/Enzo/enzo_EnzoUnits.hpp [hunk 1]
```diff
@@ -92,8 +92,9 @@ class EnzoUnits : public Units {
   /// Return temperature units scaling factor (virtual)
   virtual double temperature() const
   {
-    return (cosmology_ == NULL) ?
-      Units::temperature() : cosmology_->temperature_units();
+    double dflt_out = (enzo_constants::mass_hydrogen*std::pow(velocity(),2) /
```
Comment (gregbryan) [inline src/Enzo/enzo_EnzoUnits.hpp:95]: I think this deserves a quick explanatory comment.

**Review by gregbryan — APPROVED** [review]
Love it! I have no clever thoughts about the appropriate namespace -- enzo_constants seems reasonable to me.

**Review by jwise77 — APPROVED** [review]

**Pull request merged**